lib/Mojo/JSON.pm

```diff
@@ -0,0 +1,386 @@
+package Mojo::JSON;
```

Comment:
This is a fantastically powerful module.  Since there is support for "NotYAML" via Config, can there be a Mojo::NotYAML to do some wonderful Mojo::JSON'y things but with NotYAML?